yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -277,7 +278,7 @@ export async function generateAvmProof(
 
   // Paths for the inputs
   const bytecodePath = join(workingDirectory, 'avm_bytecode.bin');
-  const calldataPath = join(workingDirectory, 'avm_calldata.bin');
+  const calldataPath = join(workingDirectory, CALLDATA_FILENAME);
```

Comment:
If we are going to do this, let's do it with bytecode and the other filenames as well. But do prepend an AVM_CALLDATA_FILENAME, etc, because these files also deal with Noir (non-avm) contracts.

Reply:
Done.